File: python/7 kyu/438.py
# https://www.codewars.com/kata/5a5915b8d39ec5aa18000030/train/python

# Passed

from collections import Counter

def find_missing(arr1, arr2):
    arr1_counter = Counter(arr1)
    for num in arr2:
        arr1_counter[num] -= 1
        
    for num in arr1_counter:
        if arr1_counter[num] == 1:
            result = num
            break
            
    return result  

output = find_missing([4, 3, 3, 61, 8, 8], [8, 61, 8, 3, 4])
print(output)

File: nodejs/7 kyu/438.js
// https://www.codewars.com/kata/55afed09237df73343000042/train/javascript

// Passed

function isLucky(n) {
  let digits = String(n)
    .split("")
    .map((strDigit) => Number(strDigit));
  let digitsSum = digits.reduce((prev, curr) => prev + curr, 0);
  let result = digitsSum === 0 || digitsSum % 9 === 0;
  return result;
}

const output = isLucky(1892376);
console.log(output);